Multivariate/multivariate_database.ml

```diff
@@ -25,6 +25,7 @@ theorems :=
 "ABELIAN_GROUP_ISOMORPHISM_INVERSION",ABELIAN_GROUP_ISOMORPHISM_INVERSION;
 "ABELIAN_GROUP_ISOMORPHISM_POWERING",ABELIAN_GROUP_ISOMORPHISM_POWERING;
 "ABELIAN_GROUP_ISOMORPHISM_POWERING_EQ",ABELIAN_GROUP_ISOMORPHISM_POWERING_EQ;
+"ABELIAN_GROUP_ITERATE",ABELIAN_GROUP_ITERATE;
```

Comment:
Why does this PR mix changes to Group and changes to the QE parts?

Reply:
I'm not sure, it's very frustrating. 
I think it's because I did a PR from Harrison's latest update to both QE and my fork and that is causing issues...do you have any idea how to resolve this? Should I revert the PR on my repo?

Reply:
You should do a separate PR that pulls in the latest updates, and then rebase this PR onto that.